pkg/webhook/policy.go

```diff
@@ -220,7 +230,7 @@ func (h *validationHandler) getDenyMessages(res []*rtypes.Result, req admission.
 					logging.ConstraintGroup, r.Constraint.GroupVersionKind().Group,
 					logging.ConstraintAPIVersion, r.Constraint.GroupVersionKind().Version,
 					logging.ConstraintKind, r.Constraint.GetKind(),
-					logging.ConstraintAction, r.EnforcementAction,
+					logging.ConstraintAction, enforcementAction,
```

Comment:
We should leave these lines alone and just take the values from the result because of the changes suggested above.